### PR TITLE
Update type-of-receipt-ftreceiptcase.md

### PR DESCRIPTION
**Pull request by MarkusEGruber**

### CODE REVIEW

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ For Germany (DE) the country code is `0x4445`. Thus, the value of an unknown `ft
...
 | `0x4445000000000006` | **yearly-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. | [none] <br> SonstigerVorgang | 1.3- |
-| `0x4445000000000007` | **daily-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. | [none] <br> SonstigerVorgang | 1.3- |
+| `0x4445000000000007` | **daily-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. It is necessary to send this receipt at the end of each day.| [none] <br> SonstigerVorgang | 1.3- |
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:16]:
```suggestion
| `0x4445000000000007` | **daily-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. This is a zero-receipt. It is required to send this receipt at the end of each day to define the time of the accounting closure. | [none] <br> SonstigerVorgang | 1.3- |
```

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ For Germany (DE) the country code is `0x4445`. Thus, the value of an unknown `ft
...
 | `0x4445000000000005` | **monthly-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. | [none] <br> SonstigerVorgang | 1.3- |
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:14]:
```suggestion
| `0x4445000000000005` | **monthly-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. This is a zero-receipt. It is recommended to send this receipt at the end of each month to define the time of the accounting closure. | [none] <br> SonstigerVorgang | 1.3- |
```

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ For Germany (DE) the country code is `0x4445`. Thus, the value of an unknown `ft
...
 | `0x4445000000000006` | **yearly-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. | [none] <br> SonstigerVorgang | 1.3- |
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:15]:
```suggestion
| `0x4445000000000006` | **yearly-closing**<br /><br />TBD: close all open cbReceiptReference <-> Transaction-ID  <br /> <br />This receipt case works **only** with **implicit flow**. Calling without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. This is a zero-receipt. It is recommended to send this receipt at the end of each year to define the time of the accounting closure. | [none] <br> SonstigerVorgang | 1.3- |
```

**Review by AxelKutschera — APPROVED** [review]
Proposed addition

**Review by TSchmiedlechner — DISMISSED** [review]

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**